src/bp-forums/admin/tools.php

```diff
@@ -179,6 +222,24 @@ function bbp_admin_repair_list() {
 	return (array) apply_filters( 'bbp_repair_list', $repair_list );
 }
 
+/**
+ * Function get user's sub sites.
+ *
+ * @param int $user_id user id to get the sub sites
+ *
+ * @return mixed|void
+ */
+function bbp_get_blogs_list( $user_id = 0 ) {
```

Comment:
@janakkaneriya007 use https://developer.wordpress.org/reference/functions/get_sites/ here

Reply:
@janakkaneriya007 what is the use of `$user_id`? please remove not using codes.

Reply:
change the name of the function to bbp_get_sites()